vector3_for_2d_rendering.py

```diff
@@ -0,0 +1,72 @@
+"""
+renders 3d point for 2d surfaces
+"""
+
+import math, copy
+
+__version__ = "2020.9.26"
+__author__ = "xcodz-dot"
+
+AXIS_X = "x"
+AXIS_Y = "y"
+AXIS_Z = "z"
+
+
+def convert_to_2d(x: int, y: int, z: int, scale: int, distance: int) -> tuple:
+    """
+    Converts 3d point to a 2d drawable point
+
+    >>> convert_to_2d(1, 2, 3, 10, 10)
+    (7, 15)
+
+    """
+    if type(x) is not int:
```

Comment:
See PEP8 on why directly comparing types is not the recommended approach.  Also, this should raise a ValueError, not TypeError.

Reply:
ok, i will make the edits using Pycharm IDE and will avoid any PEP8 Errors